examples/seat-adjuster/AppManifest.json

```diff
@@ -1,26 +1,32 @@
-[
-    {
-        "name": "seatadjuster",
-        "vehicleModel": {
-            "src": "https://github.com/COVESA/vehicle_signal_specification/releases/download/v3.0/vss_rel_3.0.json",
-            "datapoints": [
-                {
-                    "path": "Vehicle.Cabin.Seat.Row1.Pos1.Position",
-                    "required": "true",
-                    "access": "read"
-                },
-                {
-                    "path": "Vehicle.Speed",
-                    "required": "true",
-                    "access": "read"
+{
+    "name": "seatadjuster",
+    "interfaces": [
+        {
+            "type": "vehicle-signal-interface",
+            "config": {
+                "src": "https://github.com/COVESA/vehicle_signal_specification/releases/download/v3.0/vss_rel_3.0.json",
+                "datapoints": {
+                    "required": [
+                        {
+                            "path": "Vehicle.Cabin.Seat.Row1.Pos1.Position",
+                            "access": "read"
+                        },
+                        {
+                            "path": "Vehicle.Speed",
+                            "access": "read"
+                        }
+                    ]
                 }
-            ]
+            }
         },
-        "runtime": [
-            "grpc://sdv.databroker.v1.Broker/GetDatapoints",
-            "grpc://sdv.databroker.v1.Broker/Subscribe",
-            "grpc://sdv.edge.comfort.seats.v1.Seats/MoveComponent",
```

Comment:
What's about the Seats service?

Reply:
At the moment the seat-adjuster example isn't really working, is it? We should change it to use the `vehicle-signal-interface` instead.

Reply:
It is working.
The seat-adjuster example of the SDK uses the manually created model in the examples folder, which actually supports the Seats service. 
A seat adjuster just using the VSS is part of the template.

Reply:
Note: In C++ there is no import functionality of examples from SDK into template.

Reply:
I do think we should align the examples between python and c++ logically, hence I suggest we change the seat adjuster example in here to use VSS and a generated model. gRPC interfaces will not be working until we have actually implemented the support and would leave this example broken until then.

Reply:
I can live with that. But we should then adapt the app in the C++ template also to become a skeleton (like in Python) instead of being a more complex seat adjuster example (which would be basically identical to the one here).